Lib/multiprocessing/managers.py

```diff
@@ -312,7 +315,7 @@ def fallback_repr(self, conn, ident, obj):
     def dummy(self, c):
         pass
 
-    def debug_info(self, c):
+    def debug_info(self, c): # c is unused?
```

Comment:
I don't know how multiprocessing managers work, but `c` seems to be an API convention (see `number_of_objects` below which also has an unused `c` parameter).